src/longChatPane.js

```diff
@@ -128,74 +131,139 @@ module.exports = {
+      const registrationArea = menuTable.appendChild(dom.createElement('tr'))
+      const statusArea = menuTable.appendChild(dom.createElement('tr'))
+
+      var me = UI.authn.currentUser()
+      if (me) {
+        await UI.authn.registrationControl(
+          { noun, me, statusArea, dom, div: registrationArea },
+          chatChannel,
+          mainClass
+        )
+        console.log('Registration control finsished.')
+        preferencesArea.appendChild(
+          UI.preferences.renderPreferencesForm(
             chatChannel,
-            mainClass
-          )
-          console.log('Registration control finsished.')
-          menuArea.appendChild(
-            UI.preferences.renderPreferencesForm(
-              chatChannel,
-              mainClass,
-              preferencesForm,
-              {
-                noun: 'chat room',
-                me: me,
-                statusArea: statusArea,
-                div: menuArea,
-                dom,
-                kb
-              }
-            )
+            mainClass,
+            preferencesForm,
+            {
+              noun,
+              me,
+              statusArea,
+              div: preferencesArea,
+              dom,
+              kb
+            }
           )
-        }
-      } else {
+        )
+      }
+      return preferencesArea
+    }
+
+    // @@ Split out into solid-ui
+
+    function panelCloseButton (panel) {
+      function removePanel () {
+        panel.parentNode.removeChild(panel)
+      }
+      const button =
+        UI.widgets.button(context.dom, UI.icons.iconBase + 'noun_1180156.svg', 'close', removePanel)
+      button.style.float = 'right'
+      button.style.margin = '0.7em'
+      delete button.style.backgroundColor // do not want white
+      return button
+    }
+    async function preferencesButtonPressed (_event) {
+      if (!preferencesArea) {
+        // Expand
+        preferencesArea = await renderPreferencesSidebar({ dom, noun: 'chat room' })
+      }
+      if (paneRight.contains(preferencesArea)) {
         // Close menu  (hide or delete??)
-        menuArea.parentNode.removeChild(menuArea)
-        menuArea = null
+        preferencesArea.parentNode.removeChild(preferencesArea)
+        preferencesArea = null
+      } else {
+        paneRight.appendChild(preferencesArea)
+      }
+    } // preferencesButtonPressed
+
+    //          All my chats
+    //
+    /* Build a other chats list drawer the side
+     */
+
+    function renderCreationControl (refreshTarget, noun) {
+      var creationDiv = dom.createElement('div')
+      var me = UI.authn.currentUser()
+      var creationContext = {
+        // folder: subject,
+        div: creationDiv,
+        dom: dom,
+        noun: noun,
+        statusArea: creationDiv,
+        me: me,
+        refreshTarget: refreshTarget
+      }
+      const issuePane = context.session.paneRegistry.byName('issue')
```

Comment:
**question**: Why issue pane? Should this not allow to create new *chats*?

Reply:
Good catch

Reply:
Fixed